include/zephyr/sys/clock.h

```diff
@@ -166,6 +166,49 @@ typedef struct {
 /* added tick needed to account for tick in progress */
 #define _TICK_ALIGN 1
 
+/* Maximum and minimum value TIME_T can hold (accounting for 32-bit time_t in glibc.h) */
```

Comment:
```suggestion
/* Maximum and minimum value TIME_T can hold */
```
This support any time_t not just 32bits

Reply:
Still applies